tests/python-gpu/test_gpu_updaters.py

```diff
@@ -32,6 +32,19 @@ def train_result(param, dmat: xgb.DMatrix, num_rounds: int) -> dict:
     return result
 
 
+class TestGPUUpdatersMulti:
```

Comment:
I have extracted all the multi-target/class datasets into an independent hypothesis, other than the test for CPU hist, no testing logic is changed.